public/pages/Dashboard/Container/DashboardOverview.tsx

```diff
@@ -182,58 +193,75 @@ export function DashboardOverview() {
-            selectedOptions={selectedIndices.map(buildItemOption)}
-            isClearable={true}
-            fullWidth
-          />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer />
-      <AnomaliesLiveChart
-        allDetectorsSelected={false}
-        selectedDetectors={currentDetectors}
-      />
-      <EuiSpacer />
-      <EuiFlexGroup justifyContent="spaceBetween">
-        <EuiFlexItem grow={6}>
-          <AnomaliesDistributionChart
-            allDetectorsSelected={true}
+      <DashboardHeader hasDetectors={adState.totalDetectors > 0} />
+      {isLoadingDetectors ? (
+        <div>
+          <EuiLoadingSpinner size="s" />
+          &nbsp;&nbsp;
+          <EuiLoadingSpinner size="m" />
+          &nbsp;&nbsp;
+          <EuiLoadingSpinner size="l" />
+          &nbsp;&nbsp;
+          <EuiLoadingSpinner size="xl" />
+        </div>
+      ) : adState.totalDetectors === 0 ? (
+        <EmptyDashboard />
+      ) : (
+        <Fragment>
+          <EuiFlexGroup justifyContent="flexStart" gutterSize="s">
+            <EuiFlexItem>
+              <EuiComboBox
+                id="detectorFilter"
+                placeholder={ALL_DETECTORS_MESSAGE}
+                options={getDetectorOptions(allDetectorList)}
+                onChange={handleDetectorsFilterChange}
+                selectedOptions={selectedDetectorsName.map(buildItemOption)}
+                isClearable={true}
+                fullWidth
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiComboBox
+                id="detectorStateFilter"
+                placeholder={ALL_DETECTOR_STATES_MESSAGE}
+                options={getDetectorStateOptions()}
+                onChange={handleDetectorStateFilterChange}
+                selectedOptions={selectedDetectorStates.map(buildItemOption)}
+                isClearable={true}
+                fullWidth
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiComboBox
+                id="indicesFilter"
+                placeholder={ALL_INDICES_MESSAGE}
+                options={getVisibleOptions(visibleIndices, visibleAliases)}
+                onChange={handleIndicesFilterChange}
+                selectedOptions={selectedIndices.map(buildItemOption)}
+                isClearable={true}
+                fullWidth
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer />
+          <AnomaliesLiveChart
+            allDetectorsSelected={false}
             selectedDetectors={currentDetectors}
           />
-        </EuiFlexItem>
-        <EuiFlexItem grow={3}>
-          <AnomalousDetectorsList selectedDetectors={currentDetectors} />
-        </EuiFlexItem>
-      </EuiFlexGroup>
+          <EuiSpacer />
+          <EuiFlexGroup justifyContent="spaceBetween">
+            <EuiFlexItem grow={6}>
+              <AnomaliesDistributionChart
+                allDetectorsSelected={true}
```

Comment:
looks like `allDetectorsSelected` is never used when rendering the `AnomaliesDistributionChart` in `AnomaliesDistribution.tsx`. Maybe remove this prop field entirely?

Reply:
good catch. Removed.